pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["pybind11", "setuptools", "wheel"]
```

Comment:
[PEP 518](https://peps.python.org/pep-0518/) (which defines this section of the `pyproject.toml`) [has this as the default](https://peps.python.org/pep-0518/#build-system-table):
```
requires = ["setuptools", "wheel"]
```